posthog/warehouse/types.py

```diff
@@ -1,8 +1,8 @@
-from enum import Enum
+from enum import StrEnum
 from typing import TypedDict
 
 
-class IncrementalFieldType(Enum):
+class IncrementalFieldType(StrEnum):
```

Comment:
This was the real change here! We weren't doing proper comparisons on `IncrementalFieldType` without it